### DISCUSSION TITLE
Account GitHub?

### DISCUSSION THREAD

**Discussion by ghost** [Launch-X-Latam/MisionFrontEnd#8] [Q&A]
Hola, quisiera saber si podemos usar la cuneta de github que tenemos de manera personal, o necesariamente tenemos que crearnos una cuenta con el correo institucional que nos han brindado?

Agradeceria me puedan resolver esta duda, para ver si puedo usar mi cuenta personal de Github, ya que es con la que interactuo mas.

Gracias de antemano!

**Comment by JosafatJimenezB** [comment]
Te sugiero la personal para que todos tus trabajos que realices aquí te sirvan de portafolio

**Reply by ghost** [reply]
Hola, entiendo, me podrias brindar unos pasos para realizarlo por favor :(

**Reply by JosafatJimenezB** [reply]
claro

**Reply by JosafatJimenezB** [reply]
Yo en este caso la cuenta con la que estoy comentando es mi cuenta personal, desde esta me uni a launchx y todos los trabajos mas destacados les pongo un pin para anclarlos al incio para cuando alguien vea mi perfil pueda ver mis trabajos. El logo de la organizacion tambien queda bien perfil!!

**Reply by MarcoAntonio159** [reply]
Muchas Gracias!!

**Comment by OscarTinajero117** [comment]
Agrega el correo de innovaccion a tu cuenta principal de GitHub

**Reply by ghost** [reply]
Hola, entiendo, me podrias brindar unos pasos para realizarlo por favor :(

**Reply by OscarTinajero117** [reply]
Aquí puedes ver como hacerlo

https://github.com/Launch-X-Latam/MisionFrontEnd/discussions/28

**Reply by MarcoAntonio159** [reply]
Muchas gracias Oscar!
